lib/utils/eventually-update.js

```diff
@@ -38,7 +39,7 @@ const npmUpdate = async (serverless, { newVersion, tarballUrl, abortHandler }) =
   try {
     await fse.remove(tempInstallationDir);
     await fse.ensureDir(tempInstallationDir);
-    const downloadStream = got.stream(tarballUrl);
+    const downloadStream = await fetch(tarballUrl).then((response) => response.body);
```

Comment:
`await fetch(tarballUrl).then((response) => response.body)` doesn't return stream

Again, note that this part is not covered by tests, so fact that tests pass, doesn't indicate it's a valid update

Reply:
I believe the `body` property of the node-fetch `Response` is indeed a readable stream.

Minor point: I do think it would be easier for the reader of this code if you destructured the response in this statement like you're doing in `lib/utils/standalone.js` in this same commit instead of calling `then` on the Promise that node-fetch returns.

I am dubious though about the fact that in the aforementioned file, you treat `body` as if it were a string. I'll make a separate comment about that.

Reply:
Indeed @debadutta98 sorry for confusion. Here `response.body` is a stream. Yet let's not use `then`, but rely strictly on `async/await` syntax